### PR TITLE
review code

### PR DESCRIPTION
**Pull request by Ashe546**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+        </ul> 
+        </div>  
+
+        <div >
+           <p id="ip"><i>"Borlaug's life and achievement are testimony to the far-reaching contribution that one man's towering intellect, persistence and scientific vision can make to human peace and progress."</i></p>
+           <p id="ip"><i>-- Indian Prime Minister Manmohan Singh</i></p>
+        </div>
+
+      <strong id="link">If you have time, you should read more about this incredible human being on his <a href="google.com" id="tribute-link" target="_blank"> Wikipedia entry.</a></strong>
```
Comment (Ashe546) [inline index.html:57]: You can add this to a footer section

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
-# project-review
```
Comment (Ashe546) [inline README.md:1]: you can add a more specific readme file

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,77 @@
+#main{
```
Comment (Ashe546) [inline css/style.css:1]: add spacing after class or id

**Review by Ashe546 — COMMENTED** [review]
Kindly Make the readme file more descriptive so that others can have a better understanding of your code
please Add a Linter checker as per microverse requirement.
kindly Add a description to your pull request.

File: index.html (hunk 1 above)
Comment (Ashe546) [inline index.html:57]: add this as a footer

Comment (Ashe546) [inline index.html:57]: add this as a footer

File: index.html [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    </div>
+
```
Comment (Ashe546) [inline index.html:59]: kindly Add this as a footer so that your code looks professional.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @Ashe546, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## Required Changes ♻️
- [ ] Kindly make sure to make at least two inline comments, as per [this requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/review_your_project.md#:~:text=Remember%20to%20add%20a%20minimum%20of%202%20comments).
- [ ] Please consider forming your suggestions to sound more polite and not as orders, as this will look a lot nicer for the person being reviewed. You can use words like "Please" or "Kindly" to sound more polite in your review.
- [ ] Kindly make sure to include the reason why you want this required change, as this sometimes seems confusing to some people and they require some explanation on why you think this is required.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+   
+    <div id= "main">
+     <section id="title">
+       <h1 class = "h1">Dr. Norman Borlaug</h1>
+       <p  class = "p">The man who saved a billion lives</p>
+     </section>
+     <section id="img-div">
+       <img id="image" src="image/tribute-page-main-image.jpeg">
```
Comment (Ashe546) [inline index.html:18]: Kindly add an alt attribute to specify an alternate text for an image

File: index.html [hunk 6]
```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```
Comment (Ashe546) [inline index.html:10]: please move this script tag position to before the body closing tag to make your code proffecional.

File: index.html (hunk 5 above)
Comment (zieeco) [reply index.html:18]: - [ ] **[OPTIONAL]**
- Great adding the changes however, I think you could improve more on elaborating more on the changes your propose.
- Your reason here is fair enough but not really explicit. Kindly work on this.

File: index.html (hunk 6 above)
Comment (zieeco) [reply index.html:10]: - [ ] **[OPTIONAL]**
 Great adding the changes however, I think you could improve more on elaborating more on the changes your propose.
- Your reason here is not really to make the code professional, It should be in order for the browser to load the content of the page before executing the script. Kindly, work more on your reasons.

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @Ashe546 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 
✅ Linters are passing 👏 


## Optional suggestions
_check the comments under the review_

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**